src/map/handler/Map.BoxZoom.js

```diff
@@ -113,7 +122,10 @@ export var BoxZoom = Handler.extend({
 		if (!this._moved) { return; }
 		// Postpone to next JS tick so internal click event handling
 		// still see it as "moved".
-		setTimeout(Util.bind(this._resetState, this), 0);
+		if (this._resetStateTimeout !== 0) {
```

Comment:
This is more or less exactly the same code as line 57-62; could we break this out to a private method, something like `_clearDeferredResetState`?